tools/visual-pipeline-and-platform-evaluation-tool/vippet/api/api_schemas.py

```diff
@@ -121,9 +128,13 @@ class PipelineDefinition(BaseModel):
 
 
 class PipelineValidation(BaseModel):
-    type: PipelineType
-    pipeline_description: str
-    parameters: Optional[PipelineParameters]
+    type: PipelineType = PipelineType.GSTREAMER
+    pipeline_graph: PipelineGraph
+    parameters: Optional[Dict[str, Any]] = None
```

Comment:
Maybe let's add an example to make the `max-runtime` parameter visible in the doc
```suggestion
    parameters: Optional[Dict[str, Any]] = Field(
        default=None, examples=[{"max-runtime": 10}]
    )
```